stdlib/ast.pyi

```diff
@@ -178,6 +205,10 @@ else:
     @overload
     def parse(source: str | bytes, filename: str | bytes = ..., mode: Literal["exec"] = ...) -> Module: ...
     @overload
+    def parse(source: str | bytes, filename: str | bytes = ..., mode: Literal["eval"] = ...) -> Expression: ...
+    @overload
+    def parse(source: str | bytes, filename: str | bytes = ..., mode: Literal["single"] = ...) -> Interactive: ...
+    @overload
     def parse(source: str | bytes, filename: str | bytes = ..., mode: str = ...) -> AST: ...
```

Comment:
The `= ...` here on the mode argument was also wrong and mypy I believe should have flagged it. The `= ...` should be present only for the default behavior.